scripts/start/run.R

```diff
@@ -175,7 +175,7 @@ run <- function(start_subsequent_runs = TRUE) {
   # Use the name to check whether it is a coupled run (TRUE if the name ends with "-rem-xx")
   coupled_run <- grepl("-rem-[0-9]{1,2}$",cfg$title)
   # Don't start subsequent runs form here if REMIND runs coupled. They are started in start_coupled.R instead.
-  start_subsequent_runs <- (start_subsequent_runs | isTRUE(cfg$restart_subsequent_runs)) & !coupled_run
+  start_subsequent_runs <- ! isFALSE(cfg$restart_subsequent_runs) && ! coupled_run
```

Comment:
Do I understand it correctly that if the run is restarted after preemtion the `! ifFALSE`-expression evaluates to TRUE because `cfg$restart_subsequent_runs` is NULL because it has not been set by the user in `start.R`?

Reply:
yes, if this run has not been restarted manually, it is just `NULL` and `isFALSE(NULL)` is `FALSE`, so subsequent standalone runs will be started